ClassPaulita/GuessTheCardGame.cs

```diff
@@ -91,7 +246,7 @@ public void Run()
 
         private void Play()
         {
-            Console.WriteLine($"{spacing}Play!");
+            Console.WriteLine($"{_spacing}Play!");
```

Comment:
It might be nice to say something like "Let's Play!"

Reply:
K.